src/Env.roc

```diff
@@ -37,8 +37,7 @@ exePath =
 ## Reads the given environment variable.
 ##
 ## If the value is invalid Unicode, the invalid parts will be replaced with the
-## [Unicode replacement character](https://unicode.org/glossary/#replacement_character)
-## (`�`).
```

Comment:
Is there a problem with including the actual character? 🤔

Reply:
This was previously an issue as it broke docs gen, it has since been resolved. I will add this back in.